hid/__init__.py

```diff
@@ -77,19 +80,19 @@ def enumerate(vid=0, pid=0):
 class Device(object):
     def __init__(self, vid=None, pid=None, serial=None, path=None):
         if path:
-            print 'opening with path'
+            print('opening with path')
```

Comment:
I think I would rather you go ahead and just remove all these print statements. They are all just leftover from debugging early on, and serve no real purpose in normal use.